Code2Gether-Discord-Bot.Library/BusinessLogic/LeetLogicAns.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading.Tasks;
+using Code2Gether_Discord_Bot.Library.Models;
+using Discord;
+using Discord.Commands;
+namespace Code2Gether_Discord_Bot.Library.BusinessLogic
+{
+    public class LeetAnsLogic : IBusinessLogic
+    {
+        private ILogger _logger;
+        private ICommandContext _context;
+        private string _args;
+
+        public LeetAnsLogic(ILogger logger, ICommandContext context, string args)
+        {
+            _logger = logger;
+            _context = context;
+            _args = args;
+        }
+
+        public Task<Embed> ExecuteAsync()
+        {
+            _logger.Log(_context);
```

Comment:
Since this will not be implemented here. It should either:
* Return an embed with the text "Coming Soon"
* throw a new NotImplemented exception after the .Log line